src/Compilers/CSharp/Portable/Symbols/NamespaceSymbol.cs

```diff
@@ -311,6 +311,23 @@ internal NamespaceSymbol GetNestedNamespace(NameSyntax name)
             return null;
         }
 
+        private ImmutableArray<NamedTypeSymbol> _lazyTypesWithExtensionMethods;
+        private ImmutableArray<NamedTypeSymbol> TypesWithExtensionMethods
```

Comment:
Will rename.
I considered doing this on the merged level too, but profile revealed that just this change was enough to make this no longer a problem worth pursuing compared to others.